<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogoDatosCodigoResumen</class>
 <widget class="QDialog" name="DialogoDatosCodigoResumen">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>164</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>359</width>
    <height>164</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>359</width>
    <height>164</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Datos de la Obra</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="LabelCodigo">
       <property name="text">
        <string>CÃ³digo de la obra</string>
       </property>
       <property name="buddy">
        <cstring>lineEditCodigo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCodigo">
       <property name="maxLength">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LabelResumen">
       <property name="text">
        <string>Nombre de la obra</string>
       </property>
       <property name="buddy">
        <cstring>lineEditResumen</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditResumen">
       <property name="maxLength">
        <number>50</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Completar datos</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BotonCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BotonAceptar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Crear</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BotonCancelar</sender>
   <signal>released()</signal>
   <receiver>DialogoDatosCodigoResumen</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BotonAceptar</sender>
   <signal>clicked()</signal>
   <receiver>DialogoDatosCodigoResumen</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
